src/index.js

```diff
@@ -39,13 +40,33 @@ let roomsData;
 let bookingsData;
 let user;
 let manager;
+let bookingRepo;
+
+const findToday = () => {
+  let today = new Date();
+  let dd = String(today.getDate()).padStart(2, '0');
+  let mm = String(today.getMonth() + 1).padStart(2, '0');
+  let yyyy = today.getFullYear();
+  today = yyyy + '/' + mm + '/' + dd;
+  return today;
+}
+
+let today = findToday();
+
+// ----------Get Data----------
+const updateAllData = () => {
```

Comment:
Need to revisit this.  I'm not sure if it actually makes sense to combine these.  Is it actually cleaner/more readable?